javascript/ql/lib/semmle/javascript/dataflow/TaintTracking.qll

```diff
@@ -635,39 +635,18 @@ module TaintTracking {
           pred.asExpr() = succ.getAstNode().(MethodCallExpr).getReceiver() and
           (
             // sorted, interesting, properties of String.prototype
-            name = "anchor" or
```

Comment:
for really large sets, readability suffers — like in this case. I like the consistency, though. But maybe the real issue here is the auto-formatter that won't allow having one line per item for large sets?

Reply:
+💯

One workaround we commonly is to put an empty line comment at the end of each item